docs/developer_guide/getting_started.md

```diff
@@ -14,20 +14,19 @@ Make sure you have Node.js and npm installed; you need them to obtain and use th
 
 ### Create a new Node.js project
 1. Create and enter a new directory to hold the project:
-  ### Linux/Mac
-  1. `mkdir ~/offline_wallet_generator`
-  2. `cd ~/offline_wallet_generator`
-
+  1. `$ mkdir ~/offline_wallet_generator`
+  2. `$ cd ~/offline_wallet_generator`
+<!--
   ### Windows
   1. `mkdir C:\offline_wallet_generator`
   2. `cd C:\offline_wallet_generator`
-
+-->
 2. Create the new project:
-  `npm init`
+  `$ npm init`
```

Comment:
`$ blabla` is not valid markup.
There is no need to hint it's a shell command, as it's a command only run in a shell environment.

Reply:
I made the decision to insert the '$' based on a couple other guides I was looking at as an example. I came to the conclusion - apparently prematurely - that this was just the standard convention. Looking around some more now, I see that is not the case.

Can you explain what you mean when you say it is invalid markup, though?